src/Metadata/Builder/ColumnBuilder.php

```diff
@@ -191,12 +191,16 @@ public function addConstraint(string $constraint): self
 
     private static function sanitizeName(string $name): string
     {
+        $sanitized = ColumnNameSanitizer::sanitize($name);
+
         // In some databases, eg. MsSQL is one space valid column name, so we must it sanitize to non-empty string
-        if (trim($name) === '') {
```

Comment:
`Trim` neodstrani vsetky biele znaky, iba niektore, ... `ColumnNameSanitizer` odstranuje vsetky, a tak doslo k problemu, ze to cez `trim` preslo ako "neprazdny string" ale `ColumnNameSanitizer` to vycistil uplne, a vyskocila chyba: `InvalidArgumentException:Column's sanitized name cannot be empty.`

Fixol som to a pridal test.